web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed.md

```diff
@@ -0,0 +1,22 @@
+#Guest Agent not installed
+
+The purpose of this document is to show what a VM would look like without the guest agent installed. 
+
+In the list view we show an “info” icon with the status with a tooltip that show that the guest agent isn’t installed. Further information can be provided in a tooltip.
+
+For the IP address, when the GA ins't installed, items will get “unavailable” text and a tooltip with more info on hover.
+
+![vm list](img/guest-agent-1.png)
+ 
+![vm list](img/guest-agent-2.png)
+
+![vm list](img/guest-agent-3.png)
```

Comment:
Is this our final design proposal or are we going with the one that also has the informational icon? @matthewcarleton

Reply:
My thought was to not have a repeating info icon for the unavailable IP if we already have it for the status. It seems a bit cluttered. Do you prefer it with the icon @lizsurette ?